src/views/states/list/components/utils.ts

```diff
@@ -1,20 +1,32 @@
 import { InterfaceType, NodeNetworkConfigurationInterface } from '@types';
 
-import { FILTERS_TYPES } from '../constants';
-
-export const interfaceFilters = {
-  [FILTERS_TYPES.INTERFACE_STATE]: (selectedInput, obj) => {
+import { FILTER_TYPES } from '../constants';
+import { getIPV4Address, getIPV6Address } from '@utils/interfaces/getters';
+
+export const interfaceFilters: Record<
+  string,
+  (selectedInput: string[], obj: NodeNetworkConfigurationInterface) => boolean
+> = {
+  [FILTER_TYPES.INTERFACE_STATE]: (selectedInput, obj) => {
     if (!selectedInput.length) return true;
     return selectedInput.some((status) => obj.state.toLowerCase() === status);
   },
-  [FILTERS_TYPES.INTERFACE_TYPE]: (selectedInput, obj) => {
+  [FILTER_TYPES.INTERFACE_TYPE]: (selectedInput, obj) => {
     if (!selectedInput.length) return true;
     return selectedInput.some((interfaceType) => obj.type === interfaceType);
   },
-  [FILTERS_TYPES.IP_FILTER]: (selectedInput, obj) => {
+  [FILTER_TYPES.IP_FILTER]: (selectedInput, obj) => {
     if (!selectedInput.length) return true;
     return selectedInput.some((ipType) => !!obj[ipType]);
   },
+  [FILTER_TYPES.IP_ADDRESS]: (selectedInput, obj) => {
+    const searchIpAddress = selectedInput?.[0];
```

Comment:
Sorry, I missed this one. This should be `searchIPAddress` with a capital 'P' as well.